README.md

```diff
@@ -19,7 +19,7 @@ The goal of this demo is to demonstrate a Chatbot LLM application augmented with
 
 - LLM Application augmented with content from Red Hat product documentation.
 - Multiple LLM providers (OpenAI, Hugging Face, NVIDIA)
-- Redis Vector Database to store embeddings of RedHat product documentation.
+- Vector Database, such as PGVECTOR or REDIS, to store embeddings of RedHat product documentation.
```

Comment:
What about  - PGVECTOR from EnterpriseDB (EDB) or any other distributions OR REDIS

Reply:
We will need to change the diagrams to include PGVector  , as currently it has REDIS only.

Reply:
I just mentioned PGVECTOR .. I did mention EDB where I described what the components are going to be installed